# File:        $Id$
# Revision:    $Revision$
# Author:      $Author$
# Date:        $Date$
#
# The Netarchive Suite - Software to harvest and preserve websites
# Copyright 2004-2010 Det Kongelige Bibliotek and Statsbiblioteket, Denmark
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2.1 of the License, or(at your option) any later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA

#NB encoding is set to iso-8859-1 for handling french accentuated characters.

pagetitle;monitor.summary=Vue du système
sitesection;monitor=État du système
tablefield;location=Localisation
tablefield;machine=Machine
tablefield;applicationname=Application
tablefield;applicationinstanceid=ID instance
tablefield;httpport=Port HTTP
tablefield;priority=Priorité
tablefield;replicaname=Copie de sécurité
tablefield;index=Index
tablefield;logmessage=Extrait du log
showall=afficher tout
hide=masquer
show=afficher
more.dot.dot.dot=Voir plus...
errormsg;remote.host.returned.null.log.record=Le log renvoyé par l'hôte distant est vide
errormsg;error.in.building.jmxquery=Erreur dans la construction de la requête JMX
errormsg;error.in.querying.jmx.with.query.0=Erreur dans l''exécution de la requête JMX ''{0}''
errormsg;jmx.error.while.getting.log.record=Erreur JMX lors de la récupération du log
errormsg;probably.host.is.not.responding=Il est possible que la connexion avec l'hôte distant soit temporairement perdue
errormsg;remote.jmx.bean.generated.exception=Le composant Java JMX a généré l'exception suivante :
